docs/030-qs_intro.md

```diff
@@ -54,6 +58,8 @@ To install Docker, [please see the official documentation](https://docs.docker.c
 
 ## Manual Install
 
+OVOS provides an [official installer](https://github.com/OpenVoiceOS/ovos-installer) which should allow you to easily install OVOS to a variety of OS's including Windows and MacOS.  More information on using this tool can be found on the [ovos_installer](044-ovos_installer) page in these docs.
```

Comment:
OSes/OSs/OS's is an unresolvable disagreement between style guides and editors. Best solution is to try to spell it out whenever possible; when not, my opinion is that OSes is the likeliest to go unnoticed by most readers.